app/renderer/src/components/App.js

```diff
@@ -47,6 +53,8 @@ App.defaultProps = {
 
 App.propTypes = {
   theme: ThemeSchema,
```

Comment:
is `q` missing from the prop-type validation? it is being destructed in L28, after all.